queries/boundaries.jinja2

```diff
@@ -84,6 +86,7 @@ UNION ALL
 
 SELECT
   NULL AS name,
+  0 as min_zoom,
```

Comment:
Should this kick in at zoom 8, when we switch over from Natural Earth to OpenStreetMap?